### PR TITLE
Update data-classification-content-explorer.md

### PR DESCRIPTION
**Pull request by shadansan**
Hi @chrfox 
This is per the email we discussed

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/data-classification-content-explorer.md [hunk 1]
```diff
@@ -52,6 +52,9 @@ In order to get access to the content explorer tab, an account must be assigned
 > [!IMPORTANT]
 > Membership in these role groups does not allow you to view the list of items in content explorer or to view the contents of the items in content explorer.
 
+> [!IMPORTANT]
+> only global admins can manage/assign permissions to other users in SCC. Please check this link for more details: https://docs.microsoft.com/en-in/microsoft-365/security/office-365-security/grant-access-to-the-security-and-compliance-center?view=o365-worldwide
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/data-classification-content-explorer.md]:
```suggestion
> Only Global admins can manage or assign permissions to other users in the Security & Compliance Center. For more details, see [Give users access to the Security & Compliance Center](microsoft-365/security/office-365-security/grant-access-to-the-security-and-compliance-center).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @shadansan, kindly commit the suggested change. Many thanks.

**Comment by JohanFreelancer9** [comment]
Hi @shadansan, I notice that you have closed this PR after you committed the suggested changes. Was that deliberate?

**Comment by shadansan** [comment]
Hi
Sorry I didn't know what is next and the close button was the only button available. Can you please let me know what is the process?
Best regards
Shadan

**Comment by JohanFreelancer9** [comment]
@shadansan Not a problem. You would need to re-open the PR for Chris to approve and merge.

**Comment by shadansan** [comment]
Thanks so much for the information. I re-opened it. Is there anything else I should do or Chris will see it?
Best regards
Shadan

**Comment by JohanFreelancer9** [comment]
Nothing more is needed from you at this time. Thanks so much.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by shadansan** [comment]
Thank you!

**Pull request merged**